### PR TITLE
Gj game

### PR DESCRIPTION
**Pull request by grace-joh**
## Describe your changes
- created Game class 
  - #inititalize
  - #players puts both players into an array
  - #play loops the Game methods
  - #output_start 
    - prints ASCII art 
    - asks for user input to start or quit
    - if quit, #output_goodbye
  - #output_goodbye clears the system and aborts out of the #play method
  - #take_turns loops to iterates through the #players array to 
    - #output_turn
    - let the player take their turn.move
    - break loop if there is a winner or the board is full
    - ***this loop needs attention!
  - #output_turn alerts which player's turn is next
  - #output_winner announces winner or draw
  - #output_play_again asks to play again
- added tests for all possible outputs (WIP)

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added/updated corresponding tests.

### CODE REVIEW

File: connect_four.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+________________________________________________________
+ __   __             ___  __  ___     ___  __        __
```
Comment (margdaly) [inline connect_four.txt:2]: I'm so happy we chose this! Also thank you for showing me this ruby function

File: runner.rb [hunk 2]
```diff
@@ -1 +1,4 @@
-require 'spec_helper'
```
Comment (margdaly) [inline runner.rb:1]: Good catch!

**Review by margdaly — APPROVED** [review]
We're so close!! 
Did we account for quitting the game at any time? 
and maybe prompting "are you sure? q to quit or any key to resume?

**Comment by grace-joh** [comment]
That's a good idea! I think that's something we'll have to refactor in once the actual game loops correctly... let's put it on our project board!

**Pull request merged**